src/agents/pnp/daemon/osconfig.json

```diff
@@ -27,6 +27,38 @@
       "ComponentName": "HostName",
       "ObjectName": "Hosts"
     },
+    {
```

Comment:
Possible future looking improvement to `osconfig.json`: we might want to change the `ObjectName` property to be an array of strings. Since `ComponentName` -> `ObjectName` is a one-to-many relationship it makes sense to me to structure the file in this way, this will also reduce the number of lines required to describe a component and all of its objects. For example:

```JSON
{
  "Reported": [
    {
      "ComponentName": "OsInfo",
      "ObjectNames": ["Name", "Version", "CpuType", "KernelName", "KernelRelease", "KernelVersion", "Manufacturer" "Model"]
    }
  ]
}
```

Reply:
Non-blocking and out of scope for this PR, just an idea for us to consider moving forward 😊.